src/pages/flightPage.ts

```diff
@@ -0,0 +1,207 @@
+    fromMonth : ElementFinder
+    fromMonthList : ElementArrayFinder
+    fromDay : ElementFinder
+    fromDayList : ElementArrayFinder
+    arrivingIn : ElementFinder
+    arrivingInList : ElementArrayFinder
+    returnMonth : ElementFinder
+    returnMonthList : ElementArrayFinder
+    returnDay : ElementFinder
+    returnDayList : ElementArrayFinder
+    serviceClass : string
+    airline : ElementFinder
+    airlineList : ElementArrayFinder
+    findFlightsButton : ElementFinder
+    reserveFlightButton : ElementFinder
+    passFirstName : ElementFinder
+    passLastName : ElementFinder
+    meal : ElementFinder
+    mealList : ElementArrayFinder
+    cardType : ElementFinder                        
+    cardTypeList : ElementArrayFinder               
+    creditCardNumber : ElementFinder                
+    cc_expiry_month : ElementFinder                
+    cc_expiry_month_list : ElementArrayFinder       
+    cc_expiry_year : ElementFinder                  
+    cc_expiry_year_list : ElementArrayFinder        
+    cc_first_name : ElementFinder                   
+    cc_mid_name : ElementFinder
+    cc_last_name : ElementFinder    
+    buyFlightButton : ElementFinder    
+    successMessage : ElementFinder   
+    signOfflink : ElementFinder         
+
+
+    constructor(){
+        this.tripType = 'input[name="tripType"][value="'
+        this.passengersCount = element(by.css('select[name="passCount"]'))
+        this.passengersCountList = element.all(by.css('select[name="passCount"] option'))
+        this.departingFrom = element(by.css('select[name="fromPort"]'))
+        this.departingFromList = element.all(by.css('select[name="fromPort"] option'))
+        this.fromMonth = element(by.css('select[name="fromMonth"]'))
+        this.fromMonthList =  element.all(by.css('select[name="fromMonth"] option'))
+        this.fromDay =  element(by.css('select[name="fromDay"]'))
+        this.fromDayList =  element.all(by.css('select[name="fromDay"] option'))
+        this.arrivingIn = element(by.css('select[name="toPort"]'))
+        this.arrivingInList = element.all(by.css('select[name="toPort"] option'))
+        this.returnMonth = element(by.css('select[name="toMonth"]'))
+        this.returnMonthList = element.all(by.css('select[name="toMonth"] option'))
+        this.returnDay = element(by.css('select[name="toDay"]'))
+        this.returnDayList = element.all(by.css('select[name="toDay"] option'))
+        this.serviceClass = 'input[name="servClass"][value="'
+        this.airline = element(by.css('select[name="airline"]'))
+        this.airlineList = element.all(by.css('select[name="airline"] option'))
+        this.findFlightsButton = element(by.css('input[name="findFlights"]'))
+        this.reserveFlightButton = element(by.css('input[name="reserveFlights"]'))
+
+        this.passFirstName = element(by.css('input[name="passFirst0"]'))
+        this.passLastName =  element(by.css('input[name="passLast0"]'))
```

Comment:
It's better to create a private method which will read the element based on the input id, by doing this we will delegate the responsibility to the method and if in near future change will be only in one place.